javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.gyb

```diff
@@ -0,0 +1,630 @@
+%{
+## ArraysJDK9Methods.scala.gyb
+##
+##
+## To generate this file's output manually, execute the python script
+## 'scripts/gyb.py' under the project root. For example, from the project root:
+##
+##   scripts/gyb.py \
+##     javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.gyb \
+##     --line-directive '' \
+##     -o  javalib/src/main/scala/java/util/ArraysJDK9Methods.scala.template
+##
+## Then manually fold the desired parts into Arrays.scala.
+##
+## After the manual steps, running scripts/scalafmt on the updated .scala
+## file will please CI.
```

Comment:
We can add 
```
// format: off
```
at the top of the file to disable scalafmt checks on the code generated code